app/config/ApplicationConfig.scala

```diff
@@ -41,7 +41,7 @@ class ApplicationConfig @Inject()(config: ServicesConfig){
 	lazy val reportAProblemUrl: String = s"$contactHost/contact/problem_reports"
 	lazy val reportAProblemPartialUrl: String = s"$contactHost/contact/problem_reports_ajax?service=$contactFormServiceIdentifier"
 	lazy val reportAProblemNonJSUrl: String = s"$contactHost/contact/problem_reports_nonjs?service=$contactFormServiceIdentifier"
-	lazy val signOutUrl: String = s"$signOutBaseUrl$logoutCallback"
```

Comment:
Did this old sign out url not work? I would've thought redirecting back to us on re-sign in would be preferable to a generic sign in?

Reply:
before this change, that redirected the user to ras's own feedback page (on /signed-out).

so the initial reason I removed it was because I had a look at a few other services (namely ers-checking-frontend & charities-registration-frontend) and saw that on both of them, they either redirect the user to the bas gateway (for ers) or continue on to the main gov.uk page (for charities).

I've just checked a few others, and it seems like both lisa-frontend and ers-returns-frontend actually do redirect the user to `/feedback/{service}` after signing out - will check with Sam on this one!